docs/builtins/Operators/and-q.md

```diff
@@ -1,33 +1,38 @@
 ## and?
-Use `and?` to apply a logical AND operation to the results of applying a `value` to `a` and `b`, with short-circuiting.
+
+Use `and?` to apply a logical AND operation to the results of applying a specified `value` to application functions `oper1` and `oper2`, with short-circuit evaluation.
 
 ### Basic syntax
 
-To apply logical 'and' to the results of applying `value` to `a` and `b`, use the following syntax:
+To apply a logical AND operation to the results of applying a specified `value` to the functions `oper1` and `oper2`, use the following syntax:
 
-`(and? a b value)`
+```pact
+(and? oper1 oper2 value)
+```
 
 ### Arguments
 
 Use the following arguments to specify the functions and `value` for the `and?` operation.
 
 | Argument | Type | Description |
 | --- | --- | --- |
-| `a` | `x:<r> -> bool` | Specifies the first function to apply `value` to. |
-| `b` | `x:<r> -> bool` | Specifies the second function to apply `value` to. |
-| `value` | `<r>` | Specifies the value to apply to both `a` and `b` functions. |
+| `oper1` | function | Specifies the first function to apply the specified `value` to. The result of applying the specified value returns a boolean value. |
+| `oper2` | function | Specifies the second function to apply the specified `value` to. The result of applying the specified value returns a boolean value.|
+| `value` | any | Specifies the value to apply to both `oper1` and `oper2` functions. |
```

Comment:
Ditto here, the value isn't `any`, although it will technically execute and fail at runtime, the type of the function and value are intertwined. That is to say: `oper1` and `oper2` will not accept any value `any`, the type of this function is `forall a. (a -> bool) -> (a -> bool) -> a -> bool`. So for any `a`, it applies `oper1` and `oper2` to the argument and treat it like `and (oper1 v) (oper2 v)`.

That's why the types used to be `<r>`, indicating some bound type variable, but our table format does not exactly make this easy to display.

Reply:
Maybe instead of `any` we could use a wildcard like `*` and have a convention that where you see `*` (or `any` for that matter) it means that the type depends on the function or expression being evaluated. We could even use `<r>` if it consistently means a **bound type variable** and we document that it means that. I didn't find any documentation conventions that explained this, so I wasn't convinced that we had actual conventions consistently applied.